natives.json

```diff
@@ -111574,52 +111574,52 @@
 			"build": "1207"
 		},
 		"0x5E3BDDBCB83F3D84": {
-			"name": "_0x5E3BDDBCB83F3D84",
-			"comment": "",
+			"name": "_GIVE_WEAPON_TO_PED",
```

Comment:
Since there's another GIVE_WEAPON_TO_PED native, please change this to ``_GIVE_WEAPON_TO_PED_2``.
(I'll add the info for the other one in my next commit)